app/views/exercises/show.html.haml

```diff
@@ -25,3 +26,5 @@
     .solution
       %h2 Solution
       %pre.code-highlight= highlight(@exercise.solution)
+:javascript
+  var editor = ace.edit('wrapper');
```

Comment:
This needs refactoring, basically I need to write a spec helper that will wait for the editor to initialize before trying to call it.